src/components/Admin/EditQuestion.js

```diff
@@ -0,0 +1,266 @@
+import React, { Component } from "react";
```

Comment:
The question display should just be a function component that is a form with prefilled values, so this component won't be needed after that change.

Reply:
Resolved in commit 328335366bf582d828349cde311f9f15794bbca6